app/src/main/java/org/fossasia/openevent/general/di/Modules.kt

```diff
@@ -232,6 +233,16 @@ val networkModule = module {
         objectMapper
     }
 
+    single {
+        PagedList
+            .Config
+            .Builder()
+            .setPageSize(10)
```

Comment:
@liveHarshit it is 10

Reply:
Okay. Then there are three bugs to solve that I've already told you.

Reply:
I already @iamareebjamal for a little bit of assistance as i am unable to find anything for @liveHarshit you can also suggest something if you find anything 😅

Reply:
Firstly you need to modify the logic for showing no events card and shimmer effect. Both are in conflicts right now.

Reply:
Also how the swipe refresh is connected with both.

Reply:
In a normal api call live data observer trigger when api response is made but with paged list it is triggered with a zero size list and so current implement fails and to overcome this we need and additional param that whether the intial request is completer or not

Reply:
> triggered with a zero size list

Oh, got it. So here we cannot compare the size of the list, but 

> Also how the swipe refresh is connected with both.

In that case, can we check if there are no events?

Reply:
Can only be checked after the first request is completed that is progress live data is set to false

Reply:
Then show shimmer until the request will complete.

Reply:
Also for swipe to refresh we need to clear the mutableEventsPaged but that is observable before it was a live data so its value can be set to null and I can't find anything to set it source factory to null or zero-sized list

Reply:
> Then show shimmer until the request will complete.

but that's the point I need to check 2 separate live data values and then work accordingly will find some workaround for that definitely but couldn't find anything for observable disposable.clear() would work I guess then?